src/inferenceql/query/base.cljc

```diff
@@ -44,16 +44,16 @@
   function that turns a string into a parse tree must be provided as the third
   argument.
 
-  Keywordizes the result's keys and its attributes.
+  Demunges the result's keys and its attributes.
 
   Params:
   - s - a query string
   - db - a database
   - parse - a parsing function"
   [s db parse]
   (when-let [rel (query s (atom db) parse)]
-    (let [keywordize-keys #(update-keys % query.string/safe-keyword)
-          kw-rel (map keywordize-keys rel)
-          kw-attrs (map query.string/safe-keyword (relation/attributes rel))]
+    (let [demunge-keys #(update-keys % (fn [k] (-> k str query.string/demunge)))
```

Comment:
I generally find `comp` _less_ clear than anonymous fn literals, since the symbol order is the reverse of the execution order, but since we're _already_ in an anon fn literal...yeah.